defold-rive/src/comp_rive.cpp

```diff
@@ -574,7 +482,7 @@ namespace dmRive
 
     void UpdateTransforms(RiveWorld* world)
     {
-        //DM_PROFILE(SpineModel, "UpdateTransforms");
+        //DM_PROFILE(RiveModel, "UpdateTransforms");
```

Comment:
Remove, or does this still work with ext?

Reply:
The final cleanup isn't done yet, I'd say.
There are more things to do.
FOr things like this, I've kept it in as a reminder of what to fix.